tony-core/src/test/java/com/linkedin/tony/TestTonyE2E.java

```diff
@@ -251,4 +251,26 @@ public void testTonyResourcesFlag() throws ParseException {
     int exitCode = client.start();
     Assert.assertEquals(exitCode, 0);
   }
+
+
+  /*
+   * In conditional_wait.py script, we wait for 5s for chief job type. If we only wait for worker to finish, this test
+   * is gonna fail.
+   */
+  @Test
+  public void testWaitingForChiefToFinish() throws ParseException {
+    conf.setBoolean(TonyConfigurationKeys.IS_SINGLE_NODE, false);
+    conf.setInt(TonyConfigurationKeys.TASK_MAX_MISSED_HEARTBEATS, 2);
```

Comment:
Are either of these two confs necessary? IS_SINGLE_NODE=false is already the default, and the missed heartbeats setting doesn't seem relevant to this test.

Reply:
Makes sense. Removed those 2